### PR TITLE
APIS-6954 - Accept new sandbox app commands

### PR DESCRIPTION
**Pull request by AndySpaven**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -4,14 +4,14 @@ import sbt._
...
-  val appDomainVersion = "0.35.0"
+  val apiDomainVersion = "0.16.0"
+  val commonDomainVersion = "0.13.0"
+  val appDomainVersion = "0.43.0"
 
   lazy val dependencies = Seq(
     caffeine,
     "uk.gov.hmrc"             %% "bootstrap-backend-play-30"         % bootstrapVersion,
-    "org.julienrf"            %% "play-json-derived-codecs"          % "10.1.0",
+    // "org.julienrf"            %% "play-json-derived-codecs"          % "10.1.0",
```
Comment (johnsgp) [inline project/AppDependencies.scala:14]: Can this be removed?

File: app/uk/gov/hmrc/apiplatform/modules/subscriptions/domain/services/FieldsJsonFormatters.scala [hunk 2]
```diff
@@ -68,9 +66,34 @@ trait FieldsJsonFormatters extends NonEmptyListFormatters {
...
+
+    def writes(o: ValidationRule): JsValue = o match {
+      case r: RegexValidationRule => Json.obj("RegexValidationRule" -> Json.toJson(r))
+      case u @ UrlValidationRule  => Json.obj("UrlValidationRule" -> Json.obj())
+    }
+  }
+
+  // implicit val ValidationJF: OFormat[ValidationGroup] = Json.format[ValidationGroup]
+
+  // implicit val formatValidationRule: OFormat[ValidationRule] = derived.withTypeTag.oformat[ValidationRule](ShortClassName)
```
Comment (johnsgp) [inline app/uk/gov/hmrc/apiplatform/modules/subscriptions/domain/services/FieldsJsonFormatters.scala:94]: Do we still need these?

Comment (AndySpaven) [reply app/uk/gov/hmrc/apiplatform/modules/subscriptions/domain/services/FieldsJsonFormatters.scala:94]: No

File: project/AppDependencies.scala (hunk 1 above)
Comment (AndySpaven) [reply project/AppDependencies.scala:14]: No

**Review by johnsgp — APPROVED** [review]

**Pull request merged**